src/pages/SellerUpload/ImageUploader.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { useState } from 'react';
+import axios from 'axios';
+
+import styled from 'styled-components';
+
+function ImagePreviewer() {
+  const [selectedImage, setSelectedImage] = useState([]);
+  const [ImageForUpload, setImageForUpload] = useState([]);
```

Comment:
- `컴포넌트 이름만` 대문자로 시작하니다!
```suggestion
  const [imageForUpload, setImageForUpload] = useState([]);
```

Reply:
수정했습니다.